src/assets/room_devices/presents/presents.asset.ts

```diff
@@ -0,0 +1,48 @@
+DefineRoomDeviceAsset({
+	name: 'Christmas Presents',
```

Comment:
I am a bit on the fence having a not-colorable christmas-only present room item, but the effort would be high, and christmas is soon, so let's just go with it.

We can still change it at any time in the future.